src/pages/game/game-page.jsx

```diff
@@ -20,17 +20,20 @@ export default function GamePage({ match }) {
     setGameId(match.params.gameId);
   }, [match]);
 
-  function connectWebSocket(pid, gid) {
+  function vote(points) {
+    socket.vote(points);
+  }
+
+  function addPlayer(pid, gid, cb) {
     socket.register(gid, pid, data => {
       console.log('Got a message from the server', data);
+      setPlayerId(pid);
+      setGameId(data.game.id);
       setPlayers(data.game.players);
```

Comment:
instead of keeping`data.game.players` in react's state, can we keep the whole game in there? that way we don't need to manage individual properties

Reply:
I'll open another issue for this